cmd/opm/alpha/veneer/semver.go

```diff
@@ -0,0 +1,115 @@
+package veneer
+
+import (
+	"fmt"
+	"io"
+	"io/ioutil"
+	"log"
+	"os"
+
+	"github.com/sirupsen/logrus"
+
+	// "github.com/blang/semver/v4"
+	"github.com/operator-framework/operator-registry/alpha/declcfg"
+	veneer_semver "github.com/operator-framework/operator-registry/alpha/veneer/semver"
```

Comment:
Idiomatic import strings are all lowercase concatenated words (so no underscores) and generally only with import aliases if you're importing two different packages with the same name.

```suggestion
	"github.com/operator-framework/operator-registry/alpha/veneer/semver"
```

Reply:
Yeah, looks like I was trying to resolve a situation that I thought was path resolution, when actually it was "dummy used the wrong scope, and didn't notice when he corrected that mistake while aliasing the imports". 
#OpportunitiesForHumility